parser/parser_utils.go

```diff
@@ -193,3 +194,44 @@ func parseIdentifiers(l *lexer, t token) (err error) {
 func isDMLTerminator(t token) bool {
 	return t == tkEOF || t == tkEOS || t == tkInsert || t == tkUpdate || t == tkDelete || t == tkApply
 }
+
+// PatchQueryConsistency modifies the consistency level of a QUERY message in-place
+// by locating the consistency field directly in the frame body.
+//
+// Layout based on the CQL native protocol v4 spec:
+// /* <query: long string><consistency: short><flags: byte>... */
+func PatchQueryConsistency(body []byte, newConsistency primitive.ConsistencyLevel) error {
```

Comment:
The changes are not related to parsing so this shouldn't be in `parser_utils.go`.